modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/autocomplete/Autocomplete.js

```diff
@@ -12,69 +12,62 @@
  * details.
  */
 
-import ClayAutocomplete from '@clayui/autocomplete';
-import ClayDropDown from '@clayui/drop-down';
-import {FocusScope} from '@clayui/shared';
-import {ReactPortal, useIsMounted} from '@liferay/frontend-js-react-web';
-import PropTypes from 'prop-types';
-import React, {useEffect, useRef, useState} from 'react';
+import {useIsMounted} from '@liferay/frontend-js-react-web';
+import {Autocomplete as BaseAutocomplete} from 'frontend-js-components-web';
+import React, {useEffect, useState} from 'react';
 
-import {debouncePromise} from '../../utilities/debounce';
+import debounce from '../../utilities/debounce';
 import {AUTOCOMPLETE_VALUE_UPDATED} from '../../utilities/eventsDefinitions';
-import {useLiferayModule} from '../../utilities/hooks';
 import {getData, getValueFromItem} from '../../utilities/index';
 import {showErrorNotification} from '../../utilities/notifications';
 import InfiniteScroller from '../infinite_scroller/InfiniteScroller';
 
-function Autocomplete({onChange, onItemsUpdated, onValueUpdated, ...props}) {
-	const [query, setQuery] = useState(props.initialLabel || '');
-	const [initialised, setInitialised] = useState(
-		Boolean(props.customViewModuleUrl || props.customView)
-	);
-	const [active, setActive] = useState(false);
-	const [selectedItem, updateSelectedItem] = useState(props.initialValue);
-	const [items, updateItems] = useState(null);
+function Autocomplete({
+	apiUrl,
+	autofill = false,
+	customView: CustomView,
+	disabled = false,
+	fetchDataDebounce = 200,
+	id,
+	infiniteScrollMode = false,
+	initialLabel = '',
+	initialValue = '',
+	inputClass,
+	inputId,
+	inputName,
+	inputPlaceholder = Liferay.Language.get('type-here'),
+	itemsKey,
+	itemsLabel,
+	onChange,
+	onValueUpdated,
+	pageSize = 10,
+	required,
+}) {
+	const [query, setQuery] = useState(initialLabel || '');
+	const [initialised, setInitialised] = useState(Boolean(CustomView));
```

Comment:
For some reason this caught my eye... I didn't know we were switching to british english 😂 

![queen](https://user-images.githubusercontent.com/905006/133560867-5c348b17-762c-40c2-8191-3ddf28aa385b.gif)

Reply:
Sometimes its important to pay respects